api/v1beta1/grafanadashboard_types.go

```diff
@@ -113,7 +113,7 @@ type GrafanaDashboardEnv struct {
 	Name string `json:"name"`
 	// Inline evn value
```

Comment:
typo?

Reply:
Hmm, maybe it's time for us to run a spell checker on the entire repo :sweat_smile: